src/plugins/intel_cpu/src/nodes/mha.cpp

```diff
@@ -792,8 +792,9 @@ void MHA::init_brgemm(brgemmCtx& ctx, std::unique_ptr<brgemm_kernel_t>& brgKerne
     brgemm_t brgDesc;
     brgemm_strides_t strides {static_cast<dnnl_dim_t>(ctx.M * ctx.K), static_cast<dnnl_dim_t>(ctx.K * ctx.N)};
 
+    const bool is_int8 = utils::one_of(ctx.dt_in0, data_type::u8, data_type::s8) && utils::one_of(ctx.dt_in1, data_type::u8, data_type::s8);
```

Comment:
@dmitry-gorokhov
Could you please take a look at this change.
For fp32 brgemm oneDNN expects avx512_core isa from user.
And avx512_core_vnni is expected only for s8/u8 brgemm

Reply:
The fix looks good. Thx